deltachat-ios/Controller/ChatListViewController.swift

```diff
@@ -1051,33 +1051,34 @@ extension ChatListViewController: ChatListEditingBarDelegate {
                 }
 
             let chatPresentInHomescreenWidget = allHomescreenChatsIds.contains(chatId)
-            let action: UIAlertAction
+            let action: UIAction
             if chatPresentInHomescreenWidget {
-                action = UIAlertAction(title: String.localized("remove_from_widget"), style: .default) { [weak self] _ in
+                action = UIAction(title: String.localized("remove_from_widget"), image: UIImage(systemName: "minus.square")) { [weak self] _ in
                     guard let self else { return }
                     userDefaults.removeChatFromHomescreenWidget(accountId: self.dcContext.id, chatId: chatId)
                     setLongTapEditing(false)
                 }
             } else {
-                action = UIAlertAction(title: String.localized("add_to_widget"), style: .default) { [weak self] _ in
+                action = UIAction(title: String.localized("add_to_widget"), image: UIImage(systemName: "plus.square")) { [weak self] _ in
                     guard let self else { return }
                     userDefaults.addChatToHomescreenWidget(accountId: self.dcContext.id, chatId: chatId)
                     setLongTapEditing(false)
                 }
             }
-            alert.addAction(action)
+            actions.append(action)
         }
 
         let onlyPinndedSelected = viewModel.hasOnlyPinnedChatsSelected(in: tableView.indexPathsForSelectedRows)
         let pinTitle = String.localized(onlyPinndedSelected ? "unpin" : "pin")
-        alert.addAction(UIAlertAction(title: pinTitle, style: .default) { [weak self] _ in
+        let pinImage = UIImage(systemName: onlyPinndedSelected ? "pin.slash" : "pin")
+        actions.append(UIAction(title: pinTitle, image: pinImage) { [weak self] _ in
             guard let self else { return }
             viewModel.pinChatsToggle(indexPaths: tableView.indexPathsForSelectedRows)
             setLongTapEditing(false)
         })
 
         if viewModel.hasAnyUnmutedChatSelected(in: tableView.indexPathsForSelectedRows) {
-            alert.addAction(UIAlertAction(title: String.localized("menu_mute"), style: .default) { [weak self] _ in
+            actions.append(UIAction(title: String.localized("menu_mute"), image: UIImage(systemName: "speaker.slash")) { [weak self] _ in
                 guard let self else { return }
                 MuteDialog.show(viewController: self) { [weak self] duration in
```

Comment:
This could return a sub-menu instead maybe?

Reply:
i also thouht about that, but the mute dialog is also available at other places, so it seems better to have the same UI everywhere

Reply:
Maybe those other places could use the menu too? 🤔 But can be another PR, this already addresses a lot of them.

Reply:
hm, at least whatsapp and signal also use an actionSheet for the concrete setting. i also have rarely seen a UIMenu in response to a swipe-to-action gesture (where mute is also available)

Reply:
Ah yeah I don't think swipe action is a great place for it.